lib/index.d.ts

```diff
@@ -1491,15 +1169,34 @@ declare namespace planck {
         lower(v: Vec2, w: Vec2): Vec2;
         clamp(v: Vec2, max: number): Vec2;
     }
+    let Vec3: {
```

Comment:
`Vec3` isn't in the planck namespace. The value (not the type) `planck.Vec3` is `undefined`. So you have to add it in `index.js`.